app/src/main/java/org/sagebionetworks/research/mpower/tracking/TrackingFragment.java

```diff
@@ -0,0 +1,123 @@
+package org.sagebionetworks.research.mpower.tracking;
+
+import android.arch.lifecycle.ViewModelProviders;
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.annotation.VisibleForTesting;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+import android.widget.ToggleButton;
+
+import com.google.common.base.Strings;
+
+import org.sagebionetworks.research.mpower.R;
+import org.sagebionetworks.research.mpower.tracking.TrackingViewModel.ScheduledActivityView;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import butterknife.Unbinder;
+import dagger.android.support.AndroidSupportInjection;
+
+/**
+ * This Fragment follows the dumb/passive view pattern. Its sole responsibility is to render the UI.
+ * <p>
+ * https://martinfowler.com/eaaDev/PassiveScreen.html
+ * <p>

```

Comment:
missing `</p>` so shouldn't this use `<br/>` instead?

Reply:
Javadoc is a bit odd here in allowing you to not close tags, and as of Java 8 it complains if you do use closing tags or self-closing tags
http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#examples

Reply:
ew. good to know.